servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/ConcurrentUtilsTest.java

```diff
@@ -56,13 +66,22 @@ public void tearDown() throws Exception {
     }
 
     @Test
-    public void singleThread() {
+    public void lockSingleThread() {
         assertTrue(tryAcquireLock(lockUpdater, this));
         assertTrue(releaseLock(lockUpdater, this));
     }
 
     @Test
-    public void pendingFromDifferentThread() throws Exception {
+    public void reentrantLockSingleThread() {
+        long acquireId = tryAcquireReentrantLock(reentrantLockUpdater, this);
+        assertThat(acquireId, greaterThan(0L));
+        long acquireId2 = tryAcquireReentrantLock(reentrantLockUpdater, this);
+        assertThat(acquireId2, is(-acquireId));
+        assertTrue(releaseReentrantLock(reentrantLockUpdater, acquireId, this));
```

Comment:
Add an assert for releasing `acquiredId2`?

`assertTrue(releaseReentrantLock(reentrantLockUpdater, acquireId2, this));`